pkg/meta/base.go

```diff
@@ -768,6 +797,16 @@ func (m *baseMeta) HandleQuota(ctx Context, cmd uint8, dpath string, quotas map[
 
 	switch cmd {
 	case QuotaSet:
+		format, err := m.Load(false)
+		if err != nil {
+			return errors.Wrap(err, "load format")
+		}
+		if !format.DirStats {
+			format.DirStats = true
```

Comment:
tell user the dir stats is enabled, ask them to update the stats manually, or only for current dir with quota

Reply:
achieve consensus, no need